Bad project name detection (?) with v0.7.1

When I attempt to build the master branch of https://github.com/alerque/libertinus using v0.7.1 using a local `fontship` installation, I get an error saying
```
make: *** No rule to make target '.fontship/LibertinusGitSerif-Regular-subr.otf', needed by 'LibertinusGitSerif-Regular.otf'.  Stop.
```
My guess is that the project name is inferred to be `LibertinusGit` instead of `Libertinus`.
v0.7.0 works without any problems.
(To be honest, I only suspect this to be a `fontship` issue which is why I’m reporting it here, but I’m not sure.)

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -15,7 +15,9 @@ jobs:
         run: |
           git fetch --prune --tags ||:
       - name: Setup system dependencies
-        run: sudo apt-get install fontmake fonttools libgit2-dev python3-pip psautohint python3 sfnt2woff-zopfli ttfautohint woff2 zsh
+        run: |
+          sudo apt-get update
+          sudo apt-get install fontmake fonttools libgit2-dev python3-pip psautohint python3 sfnt2woff-zopfli ttfautohint woff2 zsh
       - name: Install Rust
         uses: actions-rs/toolchain@v1
         with:

--- a/src/config.rs
+++ b/src/config.rs
@@ -32,12 +32,11 @@ impl CONF {
     pub fn from_args(&self, args: &Cli) -> Result<()> {
         if args.debug {
             self.set_bool("debug", true)?;
-        }
-        if args.quiet {
-            self.set_bool("quiet", true)?;
-        }
-        if args.verbose {
             self.set_bool("verbose", true)?;
+        } else if args.verbose {
+            self.set_bool("verbose", true)?;
+        } else if args.quiet {
+            self.set_bool("quiet", true)?;
         }
         if let Some(path) = &args.path.to_str() {
             self.set_str("path", path)?;

--- a/src/make/mod.rs
+++ b/src/make/mod.rs
@@ -36,6 +36,7 @@ pub fn run(target: Vec<String>) -> Result<()> {
         .args(&makeflags)
         .args(&makefiles)
         .args(&target);
+    let targets: Vec<_> = target.into_iter().collect();
     // Start deprecating non-CLI usage
     let gitname = status::get_gitname()?;
     let sources = status::get_sources()?;
@@ -64,7 +65,7 @@ pub fn run(target: Vec<String>) -> Result<()> {
     if CONF.get_bool("quiet")? {
         process = process.env("QUIET", "true");
     };
-    if CONF.get_bool("verbose")? {
+    if CONF.get_bool("verbose")? || targets.contains(&"debug".into()) {
         process = process.env("VERBOSE", "true");
     };
     let repo = get_repo()?;
@@ -83,7 +84,7 @@ pub fn run(target: Vec<String>) -> Result<()> {
             "FONTSHIP" => match fields[1] {
                 "PRE" => report_start(fields[2]),
                 "STDOUT" => {
-                    if fields[2] == "_gha" {
+                    if targets.contains(&"_gha".into()) {
                         println!("{}", fields[3]);
                     } else if CONF.get_bool("verbose")? {
                         report_line(fields[3]);
@@ -120,7 +121,12 @@ pub fn run(target: Vec<String>) -> Result<()> {
         Ok(ExitStatus::Exited(int)) => {
             let foo = int + ret;
             match foo {
-                0 => Ok(()),
+                0 => {
+                    if targets.contains(&"debug".into()) {
+                        dump_backlog(&backlog)
+                    };
+                    Ok(())
+                }
                 1 => {
                     dump_backlog(&backlog);
                     Err(Box::new(io::Error::new(

--- a/src/status/mod.rs
+++ b/src/status/mod.rs
@@ -48,7 +48,7 @@ pub fn get_gitname() -> Result<String> {
         let repo = get_repo()?;
         let remote = repo.find_remote("origin")?;
         let url = remote.url().unwrap();
-        let re = Regex::new(r"^(.*/)([^/]+)/?(\.git)?/?$").unwrap();
+        let re = Regex::new(r"^(.*/)([^/]+?)(/?(\.git)?/?)$").unwrap();
         let name = re
             .captures(url)
             .ok_or(Error::new("error-no-remote"))?

